README.md

```diff
@@ -277,7 +277,7 @@ Writing a new type definition is as simple as creating a new package. Start by c
 
 * **main** The entry point to the definition (canonical to "main" in NPM's `package.json`)
 * **browser** A string or map of paths to override when resolving (following the [browser field specification](https://github.com/defunctzombie/package-browser-field-spec))
-* **ambient** Denote that this definition _must_ be installed as ambient
+* **ambient** Denote that this definition should be installed as ambient to the most likely target environment
```

Comment:
Good points. Yeah, I think I was trying to cram too much of an idea into too small of a space here.

Also, I agree with the digression that "analogous" is better than "canonical".